sections/03_categorize.md

```diff
@@ -559,16 +559,37 @@ relationships in data that are important for human health but difficult to
 understand. The algorithms described in this review and others like them may
 become highly accurate and useful for various purposes, including within medical
 practice. However, to discover and avoid discriminatory applications it will be
-important to consider algorithm interpretability alongside accuracy. For
-example, if we train an algorithm to predict which drugs would be prescribed
-during a patient's visit to the doctor and there's an existing pattern of racial
-differences in prescription behavior (`TODO: @traversc - you can pick a
-different example but I think we need one - whichever one you think is most
-fully supported by the literature.`), this pattern could become baked into the
-predictions made by the algorithm. Machine learning practitioners, and
-particularly those who use deep neural networks, which are challenging to
-interpret, must remain cognizant of this possibility and make every effort to
-prevent harm from discriminatory predictions.
+important to consider algorithm interpretability alongside accuracy. A number of
+properties of genomic data will make this difficult.  First, research samples are 
+frequently non-representative of the general population of interest; they tend to 
+be sicker [@doi:10.1086/512821], more male [@doi:10.1016/j.neubiorev.2010.07.002], 
```

Comment:
Get completely what you mean by "more", it reads a bit awkward. Maybe "those with illness, males and Europeans tend to be over-represented"